tests/framework/caching/OciCacheTest.php

```
@@ -0,0 +1,74 @@
+                'expire' => 'NUMBER(10)',
+                'data' => 'BLOB',
+            ],
+        )->execute();
+    }
+
+    /**
+     * @param bool $reset whether to clean up the test database
+     *
+     * @return Connection
+     */
+    public function getConnection($reset = true)
```

PHPMD (2.15.0)

[error] The method getConnection has a boolean flag argument $reset, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)